index.js

```diff
@@ -3,7 +3,7 @@ var xtend = require('xtend');
 
 var listeningFocusTrap = null;
 
-function focusTrap(element, userOptions) {
+function createFocusTrap(element, userOptions) {
```

Comment:
Don't worry, this is backwards compatible since it is a default export.

I renamed it to be consistent with the name that is used in the demos to avoid confusion.

Reply:
not valid since I was basing the PR on v3 of focus trap